clouddriver-artifacts/clouddriver-artifacts.gradle

```diff
@@ -38,7 +38,7 @@ task unpackSdk(type: Sync) {
   // Scary but works. I think clouddriver deps in general need cleaning at some point
   // Even without the oracle bmc sdk 3rd party deps there's still multiple javax.inject and commons-X JARs
   exclude "**/*jackson*.jar"
-  exclude "**/*jersey*.jar"
+//  exclude "**/*jersey*.jar"
```

Comment:
just checking that this is tentional? if so, probably just delete this line

Reply:
not sure why I did this. I've uncommented it.